recipes/newrelic/apm/dotNet/linux-systemd.yml

```diff
@@ -22,7 +22,45 @@ processMatch: []
 
 preInstall:
   requireAtDiscovery: |
-    # Get .NET introspector tooling
+    # Check if .NET is installed
+    # There are three packages for .NET: dotnet-runtime, aspnetcore-runtime and dotnet-sdk
+    # Checking both debian and rpm based distros together
+    
+    set +e # disable fail on error
+    
+    dpkg -l dotnet-runtime* >/dev/null 2>&1
```

Comment:
@jaffinito this is a great idea. could we combine all the dpkg queries into a single call? I notice sometime that invoking a package list is not speedy, I found this to work but feel free to modify as needed "dpkg -l apt notexist notexist2 apt anothernot | grep 'apt\|notexist2\|anothernot' | wc -l" and for rhel "sudo yum -q list installed sudo.* systemd.* notexist | grep 'sudo\|notexist' | wc -l"

Reply:
Nice! Let me run some tests to see how that works.

Reply:
Hello @jaffinito just checking to see if you have some time to look into this. Let me know if you need anything. Thank You